Gopkg.toml

```diff
@@ -6,3 +6,7 @@
 [[constraint]]
   name = "github.com/prometheus/client_golang"
   branch = "master"
+
+[[constraint]]
```

Comment:
When adding a constraint or override, please include a comment giving the reason, and the conditions under which it can be removed.